apis/r/README.md

```diff
@@ -22,6 +22,12 @@ mamba install -c conda-forge -c tiledb r-tiledbsoma
 
 ## From source
 
+To install the very latest tiledbsoma development version (our `main` branch), use [`devtools::install_github()`](https://cran.r-project.org/web/packages/githubinstall/vignettes/githubinstall.html):
```

Comment:
`devtools` is midly deprecated (and pulls in too much other stuff), suggest to use `remotes::install_github()` which is a little narrower.